edx2course_axis.py

```diff
@@ -51,113 +56,121 @@
 DATADIR = "DATA"
 
 VERBOSE_WARNINGS = True
-#FORCE_NO_HIDE = True
 FORCE_NO_HIDE = False
 
-# <codecell>
-
-#-----------------------------------------------------------------------------
+LOGGER = logging.getLogger()
```

Comment:
nit (i.e. feel free to ignore, and mostly just an FYI), but we usually use `log = logging.getLogger()` with the pylint name disable, but definitely appreciate the change to using a logger :space_invader: